soccer/planning/paths/TrapezoidalPath.hpp

```diff
@@ -1,17 +1,17 @@
 #pragma once
 
-#include <optional>
+#include <protobuf/LogFrame.pb.h>
```

Comment:
What was this used for?

Reply:
It was in the file before... I can delete this though